src/backend/distributed/README.md

```diff
@@ -0,0 +1,2079 @@
+# Citus Technical Documentation 
+
+The purpose of this document is to provide comprehensive technical documentation for Citus, in particular the distributed database implementation. 
+
+# Citus Concepts 
+
+**Citus table**: A table managed by Citus through PostgreSQL hooks. Whenever the table is involved in a command, the command is handled by Citus. 
+
+**Shell table**: The “original” (inaccessible) PostgreSQL table which has been converted to a Citus table. Every node has shell tables and the corresponding Citus metadata. 
```

Comment:
"Every node has..." Should we prepend "Starting in version XX, every node has..."

Reply:
There are a lot of feature that are "since version X". I think it's best to just maintain the document in the now and provide historical context only where needed to understand the now.

Reply:
I agree. Resolving my comment thread.